Makefile

```diff
@@ -11,4 +11,7 @@ lint:
 	flake8 raiden_contracts/
 
 clean:
-	rm -rf build/ *egg-info/ raiden_contracts/data/contracts.json dist
+	rm -rf build/ *egg-info/ raiden_contracts/data/contracts.json.gz dist .eggs
+
+release: clean
+	RAIDEN_SOLC_REQUIRED=1 python setup.py sdist bdist_wheel upload
```

Comment:
It's recommended to use [`twine`](https://pypi.org/project/twine/) to upload packages instead of `python setup.py upload`

Reply:
haha I've noticed this warning when doing test upload, and I wanted create a separate issue for that. I don't have `twine` installed at the moment and I don't want to mess with keyring for passwordless auth, so I guess I'll leave it as it is now.